### PR TITLE
new recipe - ayat_tarajem_data

### PR DESCRIPTION
**Pull request by mazbrili**
quran translation data for mtafsir and elforkane

### CODE REVIEW

File: app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe [hunk 1]
```diff
@@ -0,0 +1,57 @@
+SUMMARY="Translation data from KSU-Electronic Mosshaf project"
+DESCRIPTION="Ayat data is use by other project like elforkane and mtafsir \
+get from original project to minimize maintenance .
+noted that the file is derivation from project tanzil.net"
+HOMEPAGE="http://quran.ksu.edu.sa/ayat/"
+COPYRIGHT="2014-2019 King Saud University
+           2008-2016 Tanzil.net team
+		   "
```
Comment (korli) [inline app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe]:
```suggestion
	"
```

File: app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+LICENSE="Creative Commons Attribution 3.0"
```
Comment (korli) [inline app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe]: The license text is quite different from https://creativecommons.org/licenses/by/3.0/legalcode
I would suggest another name for the license (for instance the recipe name)

File: app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	haiku
+	"
+
+BUILD_PREREQUIRES="
+	cmd:find
+	"
+
+BUILD()
+{
+
```
Comment (extrowerk) [inline app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe]: Unneeded empty line

Comment (mazbrili) [reply app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe]: ok done

File: app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe (hunk 2 above)
Comment (mazbrili) [reply app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe]: ok . change the license.. 
thanks @korli

**Comment by extrowerk** [comment]
You have a whitespace problem in the line before the BUILD_PREREQUIRES section, this is why the github test fails. Pls remove it.

**Comment by extrowerk** [comment]
Maybe it would be much easier to provide the config file as additional-file.

**Comment by mazbrili** [comment]
> You have a whitespace problem in the line before the BUILD_PREREQUIRES section, this is why the github test fails. Pls remove it.

oh i see.. thanks.. i've been fix it

**Comment by mazbrili** [comment]
> Maybe it would be much easier to provide the config file as additional-file.

oh. i don't know before it can be done like that

File: app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+ta_tamil.db,Tamil - Jan Turst Foundation
+ha_gumi.db,Hausa - Gumi
+id_indonesian.db,Indonesian - Bahasa Indonesian
+ms_basmeih.db,Malay - Basmeih
+nl_keyzer.db,Dutch - Keyzer
+sw_barwani.db,Swahili - Al-Barwani
+th_thai.db,Thai - ภาษาไทย
+uz_sodik.db,Uzbek - Мухаммад Содик
+zh_jian.db,Chinese - Ma Jian
+pr_tagi.db,فارسى - حسين تاجى كله دارى
```
Comment (Begasus) [inline app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe]: I'm guess there's a font issue on Haiku for these, in Qupzilla they show up as squeres, in Pe/Koder they don't show up at all.

Comment (mazbrili) [reply app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe]: > I'm guess there's a font issue on Haiku for these, in Qupzilla they show up as squeres, in Pe/Koder they don't show up at all.

... alter to ascii character all or any other option?

Comment (Begasus) [reply app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe]: I'm guessing it's OK to keep them as is, seeing ayat_tafasir_data has the same issue and is merged already (@korli any thoughts on this?)
@extrowerk anything that needed here?

**Pull request merged**